deploy/charts/trust-manager/templates/trust.cert-manager.io_bundles.yaml

```diff
@@ -156,6 +156,8 @@ spec:
                     description: BundleCondition contains condition information for a Bundle.
                     type: object
                     required:
+                      - lastTransitionTime
+                      - reason
```

Comment:
question: (I recognise that this would also have been added by #240) I guess this is a breaking change if anyone is setting custom conditions on their bundles, right? I'm not really scared by that, but I suppose we'd need to mention it in release notes - maybe we should update this PR's title to breaking?

I'd also be fine with leaving these as _not_ required for now, and then making them required in`v1alpha2` / `v1beta1`. I don't feel strongly either way!

Reply:
These changes are breaking for sure, but only for actors writing to `Bundle` status. For consumers of status, it is actually an improvement - since we are now promising more. I am open to any solution here, but if we want to make this change totally non-breaking the change should probably be postponed to the next API version.

Reply:
**BREAKING** looks very scary, and this won't break for users. But I am ok with it.

Reply:
I think it's mostly to remind the releaser to write a little paragraph on this in the release notes - we shouldn't make a big scary thing out of this!